packages/dev/core/src/Engines/nativeEngine.ts

```diff
@@ -595,11 +595,22 @@ export class NativeEngine extends Engine {
         // if (instancesCount) {
         //     this._gl.drawElementsInstanced(drawMode, indexCount, indexFormat, indexStart * mult, instancesCount);
         // } else {

```

Comment:
Probably good to remove these comments? Both functions.

Reply:
Removed